src/test/java/core/basesyntax/service/handlers/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.handlers;
+
+import core.basesyntax.model.Transaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static OperationStrategyImpl operationStrategyImpl = new OperationStrategyImpl();
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.BALANCE,
+                new BalanceOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.SUPPLY,
+                new SupplyOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.PURCHASE,
+                new PurchaseOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.RETURN,
+                new ReturnOperationHandler());
+    }
+
+    @Test
+    public void getBalanceOperation_Ok() {
```

Comment:
```suggestion
    public void get_balanceOperation_Ok() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.